nuxeo-compound-documents/src/test/java/org/nuxeo/compound/documents/TestCompoundDocumentImporter.java

```diff
@@ -43,6 +43,8 @@
 @RepositoryConfig(cleanup = Granularity.METHOD)
 @Deploy("org.nuxeo.ecm.platform.filemanager")
 @Deploy("org.nuxeo.ecm.platform.types")
+@Deploy("org.nuxeo.ecm.automation.core")
+@Deploy("org.nuxeo.ecm.automation.scripting")
 @Deploy("org.nuxeo.compound.documents")
```

Comment:
As we're getting more and more deploy requirements for this addon, I would suggest creating a feature for them.